web/models/campaign.ex

```diff
@@ -15,17 +15,36 @@ defmodule Jingle.Campaign do
     belongs_to :sponsor, Jingle.Sponsor
     belongs_to :podcast, Jingle.Podcast
     has_many :creatives, Jingle.Creative
-    timestamps()
+    timestamps(type: :utc_datetime)
   end
 
   @doc """
   Builds a changeset based on the `struct` and `params`.
   """
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:sponsor_id, :podcast_id, :start_date, :end_date, :due_date, :original_copy, :edited_copy, :must_say, :notes, :approved, :zone])
-    |> validate_required([:start_date, :end_date, :original_copy, :zone])
+    |> cast(utc_compatible_dates(params), [:sponsor_id, :podcast_id, :start_date, :end_date, :due_date, :original_copy, :edited_copy, :must_say, :notes, :approved, :zone])
+    |> validate_required([:start_date, :end_date, :due_date, :original_copy, :zone])
     |> assoc_constraint(:sponsor)
     |> assoc_constraint(:podcast)
   end
+
+  defp utc_compatible_dates(params) do
+    params = atomize_map(params)
+    date_params = Map.take(params, [:start_date, :end_date, :due_date])
+    parsed_date_params = for { k, v } <- date_params, into: %{}, do: { k, parse_dtim(v) }
```

Comment:
Nice.  Seems like this handles the case where not all of the date-keys are even in the params.

Reply:
exactly -- if that happens, we want the validation to catch it, not an error here